pkg/security/probe/probe.go

```diff
@@ -810,6 +810,7 @@ func (p *Probe) handleEvent(CPU int, data []byte) {
 
 	// use ProcessCacheEntry process context as process context
 	event.ProcessContext = &event.ProcessCacheEntry.ProcessContext
+	event.ProcessContext.Parent = event.ProcessContext.Ancestor.Process
```

Comment:
I would rather use a pointer and I would do set it in the `func (pc *ProcessCacheEntry) SetAncestor(...` function

Reply:
I was hesitant to set Parent every time SetAncestor() was called, because I wanted to only expose Parent at the Event root level, but I agree, moving it there makes the flow clearer.